gui/createGradebookFinalGradeFormulas.js

```diff
@@ -21,7 +21,7 @@ new QSGradebookIterator(function() {
 
     this.click("Edit it anyways");
     this.click("Add Formula");
-    $("tr.category:contains(Assignment) input").click();
+    $("tr.category:contains(Classwork) input").click();
```

Comment:
I think the default should remain at `Assignment` - seems more common to me. Thoughts?